src/Microsoft.DotNet.Build.Tasks.Feed/src/PublishArtifactsInManifestBase.cs

```diff
@@ -119,6 +119,9 @@ public abstract class PublishArtifactsInManifestBase : Microsoft.Build.Utilities
 
         private static Regex FourPartVersionRegex = new Regex(FourPartVersionPattern);
 
+        private const string SymwebServerPath = "https://microsoft.artifacts.visualstudio.com/DefaultCollection";
+
+        private const string MsdlServerPath = "https://microsoftpublicsymbols.artifacts.visualstudio.com/DefaultCollection";
```

Comment:
```suggestion
        private const string MsdlServerPath = "https://microsoftpublicsymbols.artifacts.visualstudio.com/DefaultCollection";
        
```
[nit] extra line so that it maintains the format of the file.